public/index.html

```diff
@@ -2,44 +2,21 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/logo.png" />
```

Comment:
Cool, I didn't know you could use a png. I've always converted my png logo to an ico.